src/table/settings/company_settings.ini

```diff
@@ -83,9 +83,10 @@ post_cb  = UpdateAllServiceInterval
 var      = vehicle.servint_trains
 type     = SLE_UINT16
 flags    = SF_PER_COMPANY | SF_GUI_0_IS_SPECIAL
-def      = 150
-min      = 5
-max      = 800
+max      = MAX_SERVINT_DAYS
+def      = DEF_SERVINT_DAYS_TRAINS
+min      = MIN_SERVINT_PERCENT
+interval = 1
```

Comment:
I didn't want to complain, but I am going to. Sorry :P

I don't get the order here .. max / def / min ... most others are def / min / max, which makes more sense, or min / def / max is also fine by me. But max before min feels really odd, and made me think a few times: HUH? :D

Any particular reason for this specific order?